bibtex-actions.el

```diff
@@ -481,7 +482,7 @@ TEMPLATE."
     :insert (org-cite-make-insert-processor
              #'bibtex-actions-org-cite-insert
              #'org-cite-basic--complete-style)
-    :follow (lambda (_datum _arg) (call-interactively 'bibtex-actions-at-point))))
+    :follow (lambda (_datum _arg) (call-interactively 'embark-dwim))))
```

Comment:
Is it possible to detect Embark here and use another reasonable fallback if it is not available? The implementation below also used a fallback.

Reply:
Done.

Reply:
Yes, I can add that back.

But I wanted to strip this down to first get the basics working, which it's not currently.

Reply:
Actually, I think that's why he added the at-point function; to allow people to invoke it without having embark installed.

But beyond the question you and Omar had, I was also wondering why it needed to be so complicated, with the work around comments.